lib/js/src/rpc/enums/VehicleDataType.js

```diff
@@ -97,6 +97,15 @@ class VehicleDataType extends Enum {
         return VehicleDataType._MAP.VEHICLEDATA_FUELCONSUMPTION;
     }
 
+    /**
+     * Get the enum value for VEHICLEDATA_CLIMATEDATA.
+     * @since SmartDeviceLink 7.1.0
+     * @returns {String} - The enum value.
+     */
+    static get VEHICLEDATA_CLIMATEDATA () {
+        return VehicleDataType._MAP.VEHICLEDATA_CLIMATEDATA;
+    }
+
     /**
```

Comment:
Are you able to apply this history tag to `VEHICLEDATA_EXTERNTEMP`?
   
`<history>
		<element name="VEHICLEDATA_EXTERNTEMP" since="2.0" until="X.x" />
	</history>`

I think you may just need to add `* @since SmartDeviceLink 2.0.0
     * @deprecated in SmartDeviceLink 7.1.0` 

around line 110 above `static get VEHICLEDATA_EXTERNTEMP ()`.

Reply:
It is possible, but I'm not sure we need to add something here manually. The code you see is a result of the code generator tool.